intranet/apps/eighth/views/attendance.py

```diff
@@ -598,12 +599,15 @@ def generate_roster_pdf(sched_act_ids):
 
         elements.append(Table(attendance_data, style=attendance_style, colWidths=[1.3 * inch, None, 0.8 * inch]))
         elements.append(Spacer(0, 15))
+        # NOTE: We should really not be writing raw HTML
         instructions = """
         <b>Highlight or circle</b> the names of students who are <b>absent</b>, and put an <b>"X"</b> next to those <b>present</b>.<br/>
         If a student arrives and their name is not on the roster, please send them to the <b>8th Period Office</b>.<br/>
         If a student leaves your activity early, please make a note. <b>Do not make any additions to the roster.</b><br/>
-        Before leaving for the day, return the roster and any passes to 8th Period coordinator, Catherine Massar's mailbox in the
-        <b>main office</b>. For questions, please call extension 5046 or 5078. Thank you!<br/>"""
+        Before leaving for the day, return the roster and any passes to 8th Period coordinator, {}'s mailbox in the
+        <b>main office</b>. For questions, please call extension 5046 or 5078. Thank you!<br/>""".format(
+            settings.EIGHTH_COORDINATOR_NAME
```

Comment:
Perhaps we should escape this string using one of the following:
- [`django.utils.html.format_html()`](https://docs.djangoproject.com/en/2.2/ref/utils/#django.utils.html.format_html) 
- [`html.escape()`](https://docs.python.org/3/library/html.html#html.escape)

Reply:
This should probably be incorporated in an overhaul of this report mess.

Reply:
Still, it might be a good idea to add `html.escape()` as a temporary solution.